Fix "character too large for enclosing character literal type" seen with clang
http://lists.freebsd.org/pipermail/freebsd-ports-bugs/2013-August/261138.html
--- cuneiform_src/Kern/leo/src/leo_dll.c.orig	2011-04-19 07:49:57.000000000 -0500
+++ cuneiform_src/Kern/leo/src/leo_dll.c	2015-05-29 05:20:16.000000000 -0500
@@ -622,12 +622,12 @@
 	}
 leo_current_alpha_ndx=leo_register_alphabet(alphabet);
 prn_roma_regim = (
-        !ansi_letters[(uchar)'�'] &&
+        !ansi_letters[(uchar)L'�'] &&
         ansi_letters['I'] &&
-        ansi_letters[(uchar)'�'] &&
-        ansi_letters[(uchar)'�'] &&
-        ansi_letters[(uchar)'�'] &&
-        ansi_letters[(uchar)'�'] &&
+        ansi_letters[(uchar)L'�'] &&
+        ansi_letters[(uchar)L'�'] &&
+        ansi_letters[(uchar)L'�'] &&
+        ansi_letters[(uchar)L'�'] &&
         ansi_letters['1']);
 
 if( leo_alpha_type&ALPH_R_E )
